src/App.jsx

```diff
@@ -43,6 +84,129 @@ function App() {
           <Route exact path="/question3a">
             <Question3a />
           </Route>
+          <Route exact path="/question19a">
+            <Question19a />
+          </Route>
```

Comment:
There shouldn't have been so many routes.
At the very least, you could have a single `/question/:num` with `:num` being the question's id